p2p/src/peers/mod.rs

```diff
@@ -28,6 +28,8 @@ pub struct PeerInfo {
 /// Peers TBD
 #[derive(Deserialize, Serialize)]
 pub struct Peers {
+    /// Flag whether the chain is bootstrapped, specially with regards to ice period
+    pub bootstrapped: bool,
```

Comment:
This is only set to true during actor initialization, so if the actor starts before epoch 0 it will be set to false until the node is restarted.

Reply:
True, I was missing updating upon bootstrapping!